redis-server: add page (#4356)
tldr-pages/tldr: pages/common/redis-server.md

# redis-server

> Persistent key-value database.

> > More information: <https://redis.io/>.

Reviewer: ```suggestion
> More information: <https://redis.io>.
```

> - Start redis daemon, in the foreground, with the default port (6379):

Reviewer: ```suggestion
- Start Redis server in the default port (6379) and write logs to `stdout`:
```

Author: I made a change consistent with your suggestion, but I slightly changed the english around the port number.

`redis-server`

> - Start redis daemon, as a daemon (background process), with the default port:

Reviewer: ```suggestion
- Start Redis server in the default port as a background process:
```

Author: I made a change consistent with your suggestion, but I slightly changed the english around the port number.

`redis-server --daemonize yes`

> - Start redis daemon, as a daemon, on a non-default port:

Reviewer: ```suggestion
- Start Redis server in a specific port as a background process:
```

Author: I made a change consistent with your suggestion, but I slightly changed the english around the port number.

# redis-server

> Persistent key-value database.
> More information: <https://redis.io>.

- Start Redis server, using the default port (6379), and write logs to `stdout`:

`redis-server`

- Start Redis server, using the default port, as a background process:

`redis-server --daemonize yes`

- Start Redis server, using the specified port, as a background process:

`redis-server --port {{port}} --daemonize yes`

- Start Redis server with a custom configuration file:

`redis-server {{path/to/redis.conf}}`

- Start Redis server with verbose logging:

> `redis-server --loglevel verbose`

Reviewer 2: Are there different log levels you can choose? If so, it might be worth indicating that. :+1:
```suggestion
`redis-server --loglevel {{verbose}}`
```

Reviewer: These are the options, notice is the default:
```suggestion
`redis-server --loglevel {{warning|notice|verbose|debug}}`
```

Reviewer 2: That looks better 👍🏻
